app/assets/stylesheets/directory_objects_show.css.scss

```diff
@@ -161,7 +161,19 @@ body {
   border-radius: 32px;
   color: white;
 }
+.generateQR{
```

Comment:
Match the whitespace usage here, e.g. ".generateQR {", not ".generateQR{". Same with the attributes below.

Extra whitespace generally helps readability.